index.html

```diff
@@ -12,38 +12,77 @@
     <title>Atomist | Software delivery information for cloud-native applications</title>
   </head>
   <body>
-
+      
+<!-- Collapsed Navigation -->
+      
+<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">
+    <img src="src/assets/atomist_logo_shapes_banner-home.jpg" style="position:absolute;z-index:-10;left:-1500px;">
+    <img src="src/assets/Atomist-Logo-White-Horiz.png" style="width: 110px" class="m-3">
+    <div class="container h-100 ml-0 d-flex flex-column" style="z-index:10">
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Developer</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Pricing</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Docs</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Blog</div>
+            <div id="login-collapsed" class="nav-collapsed  row-1">Log in</div>
+            <button class=" text-nowrap btn btn-primary btn-lg" onclick="window.location.href = 'https://app.atomist.com/user/signup?_ga=2.246771340.948132213.1553277995-6570700.1552576324';" id="btn-start-collapsed" >Get Started</button>
+    </div>
+</div>
+      
+<!-- Nav menu button -->
+<div class="row d-flex text-right position-fixed" style="font-size:30px;color:white;width:100%;z-index:50">
+    <div class="col-12 ml-0" onclick="toggleNav()" id="nav-button">☰</div>
+
+      <script>
+          let navVisible = false;
+          function toggleNav(){
+              let nav = document.getElementById("nav-side");
+              let navButton = document.getElementById("nav-button");
+              if (navVisible == false){
+                  console.log("nav on");
+                  nav.style.visibility = "visible";
+                  navButton.innerHTML = "×";
+                  navVisible = true;
```

Comment:
Maybe for the next sprint it'll be better to have Js in a separate file, to keep the code clean.